src/phoenix/server/api/routers/v1/experiments.py

```diff
@@ -8,6 +11,20 @@
 from phoenix.server.api.types.node import from_global_id_with_expected_type
 
 
+def _short_uuid() -> str:
+    uuid_str = str(uuid.uuid4())
+    short_uuid = hashlib.sha1(uuid_str.encode()).hexdigest()[:8]  # Take first few characters
+    return short_uuid
```

Comment:
nit
```suggestion
    return random.getrandbits(32).to_bytes(4, 'big').hex()
```

<img width="482" alt="Screenshot 2024-06-14 at 7 49 17 AM" src="https://github.com/Arize-ai/phoenix/assets/80478925/11c2a2b1-eeb3-4ee7-8313-37c0b12f8444">